### PR TITLE
Removes bottle of morphine from 5x4_tinybarbershop

### PR DESCRIPTION
**Pull request by alao3alao3**
What the fuck?

#### Changelog

:cl:    
rscdel: Removes bottle of morphine from 5x4_tinybarbershop
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Why not?

**Comment by Tipy1802** [comment]
I mean it’s morphine, not that overpowered. You are supposed to find stuff that may help you as an antag in maint, but like not anything really strong and morphine isn’t that strong and honestly you can get it rather easy from other places. I think medbay has some morphine rather publicly

**Comment by alao3alao3** [comment]
morphine can be found in and only dispensed from an upgraded sleeper, bottles of it should only be accessible from chem making it.

**Comment by SomeguyManperson** [comment]
...you don't need an upgraded sleeper just a normal one, also it can be found in medical vendors

**Comment by maxion12345** [comment]
> morphine can be found in and only dispensed from an upgraded sleeper, bottles of it should only be accessible from chem making it.

Incorrect.

Morphine is found in...

Medvendors
Medicinecabinets
Chemical cabinets
A table in medbay storage.
and maint apparently


tl;dr, morphine bottles can be found in a surprisingly large amount of places, and its even available for purchase from a vendor.

**Comment by alao3alao3** [comment]
aight

**Comment by Moja10** [comment]
What Maxion said, it's really not a big deal IMO

**Comment by alao3alao3** [comment]
what other maint areas have morphine in them? I really think it should just be in medical and medical related areas, but I'll put that on those reviewers.

**Comment by patpol4** [comment]
> what other maint areas have morphine in them? I really think it should just be in medical and medical related areas, but I'll put that on those reviewers.

Maps are text files, you can search the entire map folder for morphine bottles and then look for them in the editor, i recommend VSC for searching entire folders for strings in files.

**Comment by patpol4** [comment]
Closing due to this not being that big of a deal, if it was a toxin then yeah, but morphine is not that harmful or rare.
